protos/gen/service.pb.go

```diff
@@ -1,30 +1,35 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.23.0
+// 	protoc        v3.15.4
 // source: service.proto
 
-package datacatalog
+package gen
```

Comment:
does this just work? you don't need to update any imports?

Reply:
oops that looks wrong